File: android-studio-projects/Update Tile/wear/src/main/java/com/crazyj36/updatetile/MyAccessibilityService.java
package com.crazyj36.updatetile;

import android.accessibilityservice.AccessibilityService;
import android.accessibilityservice.GestureDescription;
import android.graphics.Path;
import android.util.Log;
import android.view.accessibility.AccessibilityEvent;

import org.jetbrains.annotations.Nullable;

public class MyAccessibilityService extends AccessibilityService {

   public void onAccessibilityEvent(@Nullable AccessibilityEvent p0) {
      if (p0 != null && p0.getEventType() == AccessibilityEvent.TYPE_VIEW_CLICKED) {

      }

   }

   public void onInterrupt() {
      Log.d("UPDATETILE", "onInterrupt()");
   }
   public void pressScreen() {

   }
   protected void onServiceConnected() {
      super.onServiceConnected();
      Log.d("UPDATETILE", "onServiceConnected()");
   }
}

File: android-studio-projects/Update Tile/wear/src/main/java/com/crazyj36/updatetile/MyAccessibilityService.kt
package com.crazyj36.updatetile

import android.accessibilityservice.AccessibilityService
import android.accessibilityservice.GestureDescription
import android.graphics.Path
import android.util.Log
import android.view.accessibility.AccessibilityEvent

class MyAccessibilityService : AccessibilityService() {

    override fun onAccessibilityEvent(event: AccessibilityEvent?) {
        Log.d("UPDATETILE", "onAccessibilityEvent()")
        Log.d("UPDATETILE", event.toString())
        /*MyTileService.count++
        val path = Path()
        val gestureBuilder: GestureDescription.Builder = GestureDescription.Builder()
        path.moveTo(220f, 200f)
        gestureBuilder.addStroke(GestureDescription.StrokeDescription(path, 0, 100))
        val gesture = gestureBuilder.build()
        Log.d("UPDATETILE", "dispatched: " + dispatchGesture(gesture, null, null).toString())*/
    }

    override fun onInterrupt() {
        Log.d("UPDATETILE", "onInterrupt()")
    }
}